yarn.lock

```diff
@@ -565,11 +565,6 @@
   dependencies:
```

Comment:
Not sure we need lock file in this pr

Reply:
it's after yarn install, anyway someone will merge this sooner or later :)

Reply:
looks like it's better to commit yarn.lock https://stackoverflow.com/questions/39990017/should-i-commit-the-yarn-lock-file-and-what-is-it-for